### PR TITLE
Migrate hopper articles

### PR DESCRIPTION
**Pull request by Southpaw1496**
I'm going through the process of migrating most of the articles from the `master` branch into the `jekyll` branch. Some of them won't make it because they're short enough to be factoids.

### CODE REVIEW

File: _help/modifer-name/index.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+6. Press `Esc` to bring up the pause menu
+7. Click "Open to LAN"
+8. Click the "Allow Cheats: OFF" button until it says "Allow Cheats: ON"
+9. Click "Start LAN World"
+Type `/clear @p` into the chat again
+
+#### If the item is in your inventory
+
+1. If your world is open, save and quit to the title screen
+2. You will need an NBT editor, such as [NBTExplorer]((http://www.minecraftforum.net/topic/840677-nbtexplorer-nbt-editor-for-windows-and-mac/)
```
Comment (urielsalis) [inline _help/modifer-name/index.markdown:30]: Link seems broken

Comment (O5X) [reply _help/modifer-name/index.markdown]: Link is working fine for me

Comment (urielsalis) [reply _help/modifer-name/index.markdown]: it is now, it had 2 (

**Comment by LordRalex** [comment]
Build Warning: Layout 'article' requested in _help/could-not-create-game-directory/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/dxdiag/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/failed-check-session-lock/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/failed-to-load-skull/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/finding-minecraft-data-folder/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/guides/changing-game-directory/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/guides/drivers/amd-manual-install/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/guides/drivers/intel-manual-install/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/guides/getting-minecraft-logs/index.md does not exist.
     Build Warning: Layout 'article' requested in _help/how-to-reinstall-java/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/incompatible-class-change-error/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/internet-crash-att2wire-02/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/loadlibrary-error-126/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/modifer-name/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/pixel-format-not-accelerated/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/unable-fit-texture/index.markdown does not exist.
     Build Warning: Layout 'article' requested in _help/using-event-viewer/index.markdown does not exist.

**Comment by LordRalex** [comment]
No images load either.

**Comment by Southpaw1496** [comment]
I have absolutely zero idea why none of them exist, they work perfectly for me and the images are all there.

**Comment by Southpaw1496** [comment]
Oh and I was given the header template by uriel and have been copy-pasting it so that's not the problem

**Pull request merged**